get-started/part2.md

```diff
@@ -82,16 +86,19 @@ ENV NAME World
 CMD ["python", "app.py"]
 ```
 
-This `Dockerfile` refers to a couple of things we haven't created yet, namely
-`app.py` and `requirements.txt`. Let's get those in place next.
+This `Dockerfile` refers to a couple of files we haven't created yet, namely
+`app.py` and `requirements.txt`. Let's create those next.
 
 ## The app itself
 
-Grab these two files and place them in the same folder as `Dockerfile`.
-This completes our app, which as you can see is quite simple. When the above
-`Dockerfile` is built into an image, `app.py` and `requirements.txt` will be
-present because of that `Dockerfile`'s `ADD` command, and the output from
-`app.py` will be accessible over HTTP thanks to the `EXPOSE` command.
+Let's create two more files, `requirements.txt` and `app.py`, and put them in
+the same project folder with the `Dockerfile`. This will complete our app, which
+as you can see is quite simple. When the above `Dockerfile` is built into an
+image, `app.py` and `requirements.txt` will be present because of `Dockerfile`'s
+`ADD` command, and the output from `app.py` will be accessible over HTTP thanks
+to the `EXPOSE` command.
+
+The contents for these two new files is given below.
```

Comment:
Probably don't need this if you front-load the copy/paste information in the previous paragraph